backend/src/openarchiefbeheer/accounts/api/views.py

```diff
@@ -9,6 +9,38 @@
 from .serializers import UserSerializer
 
 
+@extend_schema(
+    tags=["Users"],
+    summary=_("Users list"),
+    description=_("List all the users."),
+    responses={
+        200: UserSerializer(many=True),
+    },
+)
+class UsersView(ListAPIView):
+    serializer_class = UserSerializer
+
+    def get_queryset(self) -> QuerySet[User]:
+        return User.objects.all()
```

Comment:
Is there a reason for not doing:

```suggestion
class UsersView(ListAPIView):
    serializer_class = UserSerializer
    queryset = User.objects.all()
```

Reply:
- For endpoints using the manager method that filter on permissions, this may cause issues when initializing as the queryset is initialized straight away.
- For the this particular case its uses the get_queryset method for consistency.